src/pages/HttpBin/index.js

```diff
@@ -49,20 +38,18 @@ const useStyles = makeStyles((theme) => ({
 
 const HttpBin = (props) => {
   const classes = useStyles();
-  const {
-    httpbin = {},
-    common: { init = false, error = null },
-  } = props;
+  const { httpbin = {} } = props;
 
   const [methodType, setCurrency] = useState('GET');
   const [queryParameters, setQueryParameters] = useState('');
   const [bodyParameters, setBodyParameters] = useState('');
   const [response, setResponse] = useState({});
   const [responseIsLoad, setResponseIsLoad] = useState(true);
 
-  if (init) {
-    props.fetchHttpBinData();
-  }
+  useEffect(() => {
+    props.fetchFirebaseData();
```

Comment:
Why do you need to fetch firebase data here?

Reply:
You're right, I don't need to use fetchFirebaseData here. Updated in last commit